test/Grpc.IntegrationTests/OrchestrationPatterns.cs

```diff
@@ -270,7 +270,7 @@ public async Task ActivityFanOut()
                     Array.Reverse(results);
                     return results;
                 })
-                .AddActivity<object, string>(toStringActivity, (ctx, input) => input!.ToString()));
+                .AddActivity<object, string>(toStringActivity, (ctx, input) => input.ToString()!));
```

Comment:
Why is the `!` needed here, after the `ToString()` call?

Reply:
Two changes are happening at once here:

`TInput?` -> `TInput`: this allows for the change of `input!` -> `input`.
`TOutput?` -> `TOutput`: `object.ToString()` has a return type of `string?`, so I have to add the `!` at the end.

Reply:
I'm just trying to understand whether users will need to add `!` to their code more often with this change. Could you have alternatively declared this as `.AddActivity<object, string?>(...)` to avoid the use of `!` here?

Reply:
>I'm just trying to understand whether users will need to add ! to their code more often with this change. 

To answer this, no they won't. This change gives users the control over nullable or not, and so they can choose the one that best fits their use case. Before it was always `TInput?`, so users were forced into the nullable scenario (and so they may have to use `!` more often then).